### DISCUSSION TITLE
Adding another custom dice term

### DISCUSSION THREAD

**Discussion by QckSilverDragon** [Muttley/foundryvtt-fallout#523] [Q&A] [answered]
Ok, so im trying to make a new location roller just for vehicles.
I copied DieFalloutLocation.mjs and renamed DieFalloutVehicleLocation.mjs and renamed the class.
I changed the DENOMINATION to "vh"

In initHook.mjs i added an import for my new class.
In the registerDiceSetttings i added CONFIG.Dice.terms.vh = DieFalloutVehicleLocation;

When i try /r 1dvh in chat i get the below error.

Am i missing some register or setting somewhere else?

![image](https://github.com/user-attachments/assets/ba7a0895-b2f6-444a-b60f-090d0472e5f9)

the console shows the term is registered.
![image](https://github.com/user-attachments/assets/04da29ca-1269-48d5-be71-7db2fb889c74)

**Comment by Muttley** [comment, marked as answer]
I really wouldn't bother with another custom die.  There's far too much variability with the vehicle locations anyway, so Rolltables would be the better choice for now.

Later on I plan switching out the current hit location system for something else.

**Reply by QckSilverDragon** [reply]
How do you edit the compendiums? I want to add more rolltables but all i see are json files and i dont know how to manage these.

**Reply by Muttley** [reply]
> How do you edit the compendiums? I want to add more rolltables but all i see are json files and i dont know how to manage these.

Please hold off on that.  I'd need to get permission from Modiphius to add anything extra to the compendiums.

**Reply by QckSilverDragon** [reply]
Ok, i was just adding rolltable hit locations for the vehicles. not adding the vehicles themselves.

P.S. just noticed the hit location displayed for normal weapon rolls also includes the number it rolled so you can reference the vehicle sheet for the location easily.
